Tools/Scripts/webkitpy/api_tests/runner.py

```diff
@@ -146,10 +146,25 @@ def run(self, tests, num_workers):
 
             with TaskPool(
                 workers=self._num_workers,
+                mutually_exclusive_groups=list(self.port.sharding_groups(suite='api-tests').keys()),
```

Comment:
You are going to want to save this value and operate differently if it's not defined (that's particularly relevant for non-Apple platforms)